internal/handler/web/projects.go

```diff
@@ -171,6 +169,59 @@ func (h *Handler) GetProjectsTreProjectId(ctx *gin.Context, projectId string) {
 	ctx.JSON(http.StatusOK, response)
 }
 
+func (h *Handler) PutProjectsTreProjectId(ctx *gin.Context, projectId string) {
+	projectUUID, err := parseUUIDOrSetError(ctx, projectId)
+	if err != nil {
+		return
+	}
+
+	projectUpdateData := openapi.ProjectTREUpdate{}
+	if err := bindJSONOrSetError(ctx, &projectUpdateData); err != nil {
+		return
+	}
+
+	// Get the existing project to validate study ownership
+	projectTRE, err := h.projects.ProjectTreById(projectUUID)
+	if err != nil {
+		setError(ctx, err, "Failed to get project")
+		return
+	}
+
+	if projectTRE == nil {
+		ctx.Status(http.StatusNotFound)
+		return
+	}
```

Comment:
Is this check needed? Or is it already covered by the RBAC? Can remove if so (same for `PostProjectsTre`)